src/pkgx.sh/Hero.tsx

```diff
@@ -1,67 +1,113 @@
 import { Box, InputAdornment, Button, TextField, Typography, Stack, Snackbar, Alert, Tooltip, useMediaQuery, useTheme, Tabs, Tab } from "@mui/material";
-import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
-import ContentCopyIcon from '@mui/icons-material/ContentCopy';
+import ArrowForwardIcon from "@mui/icons-material/ArrowForward";
+import ContentCopyIcon from "@mui/icons-material/ContentCopy";
 import React, { useState } from "react";
-import HeroTypography from "../components/HeroTypography"
+import HeroTypography from "../components/HeroTypography";
+import { useSearchParams } from "react-router-dom";
 
 export default function Hero() {
   const theme = useTheme();
-  const isxs = useMediaQuery(theme.breakpoints.down('md'));
+  const isxs = useMediaQuery(theme.breakpoints.down("md"));
+  const [searchParams, setSearchParams] = useSearchParams({ tab: "brew" });
 
-  const [open, setOpen] = useState(false)
-  const close = () => setOpen(false)
-  const [tabIndex, setTabIndex] = useState(0)
+  const [open, setOpen] = useState(false);
+  const close = () => setOpen(false);
 
-  const text = () => tabIndex == 0 ? "brew install pkgxdev/made/pkgx" : "curl -Ssf https://pkgx.sh | sh"
+  const text = () =>
+    searchParams.get("tab") === "brew"
+      ? "brew install pkgxdev/made/pkgx"
+      : "curl -Ssf https://pkgx.sh | sh"
 
   const click = (event: React.MouseEvent<HTMLElement>) => {
-    navigator.clipboard.writeText(text())
-    setOpen(true)
+    navigator.clipboard.writeText(text());
```

Comment:
there's still a bunch of unrelated changes in here, like adding semi-colons and changing quote types. Please remove

Reply:
I noticed that there are semicolons in the pre-existing code. I'm making a mental note to open a PR to fix those in the future... maybe in a cleanup PR

Reply:
I'm sure there are as the editor adds them. Which is bad of me to commit them, but I'm moving fast.

These directions here are more about PR ettiquete. I have to review every line you submit, so when you add diff lines that are meaningless you are imposing more load on me and I may make mistakes.

Diffs should be minimal to ensure reviews are as correct as possible.